File: sources/server/app/models/entities/CarClass.scala
package models.entities

import java.sql.Date

case class CarClass(
 id: Long,
 name: String,
 rate: Double,
 comment: Option[String],
 creationDate: Option[Date],
 editDate: Option[Date],
 creatorId: Option[Long],
 editorId: Option[Long]
) extends Entity

File: sources/client/app/models/car-class.js
import DS from "ember-data";

let attr = DS.attr;
var CarClassModel = DS.Model.extend({
  name: attr("string"),
  rate: attr("number"),
  comment: attr("string"),
  creationDate: attr("date"),
  editDate: attr("date"),
  creator: DS.belongsTo("user", {inverse: null, async: true}),
  editor: DS.belongsTo("user", {inverse: null, async: true})
});

export default CarClassModel;
